### PR TITLE
follow ja:glossary/controller.md v1.17

### LINKED ISSUE #21297: ja: Make glossary/controller.md follow v1.17 of the original text
**This is a Feature Request**

**What would you like to be added**
Update glossary/controller.md to follow v1.17 of the original (Englist) text.

**Why is this needed**
content/ja/docs/reference/glossary/controller.md it outdated.

**Comments**
/language ja
/good-first-issue

File to update
https://github.com/kubernetes/website/blob/dev-1.17-ja.2/content/ja/docs/reference/glossary/controller.md

Original
https://github.com/kubernetes/website/blob/release-1.17/content/en/docs/reference/glossary/controller.md

diff between translated and v1.17
https://gist.github.com/oke-py/d7abfde08cdca558165ee46e8b44c95a

**Note**
Currently, we use `dev-1.17-ja.2` branch. Please open a PR targeting the branch.

ref
[How To Contribute](https://kubernetes-docs-ja.kibe.la/shared/entries/c5878aa5-ad1f-4f29-a5bb-25853cbc14ec)
[翻訳スタイルガイド](https://kubernetes-docs-ja.kibe.la/shared/entries/5efe4fa7-d2a1-4a1d-8bc3-ce7ccdc064a6)

If you have a question, feel free to ask us here or slack.k8s.io #kubernetes-docs-ja channel.

### PR DESCRIPTION
**Pull request by translucens**
Close #21297

### CODE REVIEW

File: content/ja/docs/reference/glossary/controller.md [hunk 1]
```diff
@@ -11,8 +11,11 @@ tags:
 - architecture
 - fundamental
 ---
- クラスターの状態を{{< glossary_tooltip text="apiserver" term_id="kube-apiserver" >}}から取得して監視する制御ループで、現在の状態を望ましい状態に移行するように更新します。
+Kubernetesでは、コントローラーは{{< glossary_tooltip term_id="cluster" text="cluster" >}}の状態を監視し、必要に応じて変更を加えたり要求したりする制御ループです。それぞれのコントローラーは現在のクラスターの状態を望ましい状態に近づけるように動作します。
```
Comment (inductor) [inline content/ja/docs/reference/glossary/controller.md:14]:
```suggestion
Kubernetesにおいて、コントローラーは{{< glossary_tooltip term_id="cluster" text="cluster" >}}の状態を監視し、必要に応じて変更を加えたり要求したりする制御ループです。それぞれのコントローラーは現在のクラスターの状態を望ましい状態に近づけるように動作します。
```

File: content/ja/docs/reference/glossary/controller.md [hunk 2]
```diff
@@ -11,8 +11,11 @@ tags:
...
 
 <!--more--> 
 
-現在Kubernetesに同梱されているコントローラーの例には、レプリケーションコントローラー、エンドポイントコントローラー、名前空間コントローラー、およびサービスアカウントコントローラーがあります。
+コントローラーはクラスターの状態を{{< glossary_tooltip term_id="control-plane" >}}の一部である{{< glossary_tooltip text="apiserver" term_id="kube-apiserver" >}}から取得します。
```
Comment (inductor) [inline content/ja/docs/reference/glossary/controller.md:18]:
```suggestion
コントローラーはクラスターの状態を{{< glossary_tooltip term_id="control-plane" text="コントロールプレーン" >}}の一部である{{< glossary_tooltip text="kube-apiserver" term_id="kube-apiserver" >}}から取得します。
```

File: content/ja/docs/reference/glossary/controller.md [hunk 3]
```diff
@@ -11,8 +11,11 @@ tags:
...
+
+いくつかのコントロールプレーン内部で動くコントローラーは、Kubernetesの主要な操作に対する制御ループを提供します。
+例えば、Deploymentコントローラー、Daemonsetコントローラー、Namespaceコントローラー、Persistent Volumeコントローラー等は{{< glossary_tooltip term_id="kube-controller-manager" >}}の内部で動作します。
```
Comment (inductor) [inline content/ja/docs/reference/glossary/controller.md:21]:
```suggestion
例えば、Deploymentコントローラー、Daemonsetコントローラー、Namespaceコントローラー、Persistent Volumeコントローラー等は{{< glossary_tooltip term_id="kube-controller-manager" test="kube-controller-manager" >}}の内部で動作します。
```

Comment (translucens) [reply content/ja/docs/reference/glossary/controller.md:21]:
```suggestion
例えば、Deploymentコントローラー、Daemonsetコントローラー、Namespaceコントローラー、Persistent Volumeコントローラー等は{{< glossary_tooltip term_id="kube-controller-manager" text="kube-controller-manager" >}}の内部で動作します。
```

Comment (inductor) [reply content/ja/docs/reference/glossary/controller.md:21]: oops :P

File: content/ja/docs/reference/glossary/controller.md [hunk 4]
```diff
@@ -11,8 +11,11 @@ tags:
...
 ---
- クラスターの状態を{{< glossary_tooltip text="apiserver" term_id="kube-apiserver" >}}から取得して監視する制御ループで、現在の状態を望ましい状態に移行するように更新します。
+Kubernetesにおいて、コントローラーは{{< glossary_tooltip term_id="cluster" text="cluster" >}}の状態を監視し、必要に応じて変更を加えたり要求したりする制御ループです。それぞれのコントローラーは現在のクラスターの状態を望ましい状態に近づけるように動作します。
 
 <!--more--> 
 
-現在Kubernetesに同梱されているコントローラーの例には、レプリケーションコントローラー、エンドポイントコントローラー、名前空間コントローラー、およびサービスアカウントコントローラーがあります。
+コントローラーはクラスターの状態を{{< glossary_tooltip term_id="control-plane" text="コントロールプレーン" >}}の一部である{{< glossary_tooltip text="kube-apiserver" term_id="kube-apiserver" >}}から取得します。
+
+いくつかのコントロールプレーン内部で動くコントローラーは、Kubernetesの主要な操作に対する制御ループを提供します。
```
Comment (inductor) [inline content/ja/docs/reference/glossary/controller.md:20]:
```suggestion
コントロールプレーン内部で動くいくつかのコントローラーは、Kubernetesの主要な操作に対する制御ループを提供します。
```

File: content/ja/docs/reference/glossary/controller.md [hunk 5]
```diff
@@ -11,8 +11,11 @@ tags:
 - architecture
 - fundamental
 ---
- クラスターの状態を{{< glossary_tooltip text="apiserver" term_id="kube-apiserver" >}}から取得して監視する制御ループで、現在の状態を望ましい状態に移行するように更新します。
+Kubernetesにおいて、コントローラーは{{< glossary_tooltip term_id="cluster" text="cluster" >}}の状態を監視し、必要に応じて変更を加えたり要求したりする制御ループです。それぞれのコントローラーは現在のクラスターの状態を望ましい状態に近づけるように動作します。
```
Comment (oke-py) [inline content/ja/docs/reference/glossary/controller.md:14]:
```suggestion
Kubernetesにおいて、コントローラーは{{< glossary_tooltip term_id="cluster" text="クラスター" >}}の状態を監視し、必要に応じて変更を加えたり要求したりする制御ループです。それぞれのコントローラーは現在のクラスターの状態を望ましい状態に近づけるように動作します。
```

**Pull request merged**